src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerProvider.cs

```diff
@@ -153,7 +153,7 @@ private void UpdateFormatterOptions(ConsoleFormatter formatter, ConsoleLoggerOpt
             {
                 defaultFormatter.FormatterOptions = new SimpleConsoleFormatterOptions()
                 {
-                    DisableColors = deprecatedFromOptions.DisableColors,
+                    ColorBehavior = deprecatedFromOptions.DisableColors ? LoggerColorBehavior.Disabled : LoggerColorBehavior.Enabled,
```

Comment:
Should this be `LoggerColorBehavior.Enabled` or `LoggerColorBehavior.Default` in the else case?

Reply:
I think xunit tests are also an example of when console is redirected and in that case we are not disabling. So I prefer for old behavior to remain as Enabled.

Reply:
I don't really like using unit tests for reasoning on why an API behaves how it does. I think we should make the API work how we think our customers would expect it to work.

I don't have a super strong opinion here. Since this API is `Obsolete`, keeping it working like it did in the past makes sense.